recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2018.04.19" %}
+{% set version = "2018.04.26" %}
```

Comment:
please don't change the date.  That is a release-time activity.  If you change it here, it makes merge conflicts much more likely.

Reply:
I agree, but for this case, I was thinking of making a release with this change. What do you think?

Reply:
Yeah, go ahead. There is only 1 PR pending and that is yours :smile: